api/RTCPeerConnection.json

```diff
@@ -211,24 +205,12 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/addIceCandidate",
           "spec_url": "https://w3c.github.io/webrtc-pc/#dom-peerconnection-addicecandidate",
           "support": {
-            "chrome": [
-              {
-                "version_added": "51",
-                "notes": "Promise-based version."
```

Comment:
These notes aren't a mistake, `addIceCandidate()` did change from callback-based to promise-based. I'd suggest a `partial_implementation` range for the callback variant.

Reply:
The spec had actually made a change from callbacks to promise-based syntax (as suggested in https://groups.google.com/a/chromium.org/g/blink-dev/c/V7-02zJiVzw), so maybe it would be better to create a subfeature for these, `returns_promise`?  (There's no data guideline for this, just an open discussion in #4468, but a quick search in BCD shows this is the more common way to represent the transition.  This file is also the only one with that `with_callback` subfeature so we can reverse it and turn it into `returns_promise`.)

Reply:
I've now noticed your PR to make the name of the subfeatures consistent in #11630!

Reply:
Sure, using `returns_promise` subfeatures sounds OK. It does make it more prominent, and I suspect supported this from the beginning, but I think it'll be OK.

Reply:
All done, and ready for re-review!